package.json

```diff
@@ -5,9 +5,9 @@
   "main": "./lib/index.js",
   "scripts": {
     "lint": "eslint .",
-    "prepublish": "babel --out-dir ./lib ./src",
+    "prepublish": "NODE_ENV=development babel --out-dir ./lib ./src",
```

Comment:
I had an issue with `babel`, not sure if it's just my machine, but it was complaining that I needed to pass an environment. I can look more into this.